src/compat/posix/time/strptime.c

```diff
@@ -0,0 +1,302 @@
+/**
+ * @file strptime.c
+ * @brief writing linux function strptime for embox os
+ *
+ * author Tamara Deryugina <tamara.deryugina@gmail.com>
+ * @date 02.03.2020
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+#include <ctype.h>
+#include <math.h>
+
+static int num_length(char *new_s) {
+	int count = 0;
+	while ((new_s[count] >= '0') && (new_s[count] <= '9')) {
+		count++;
+	}
+	return count;
+}
+
+static int word_length(const char *s) {
+	int count = 0;
+	while ((s[count] >= 'a') && (s[count] <= 'z')) {
+		count++;
+	}
+	return count;
+}
+
+static int degree(int dec, int num) {
```

Comment:
Do you really need `int dec` argument?

Reply:
Mm, you're right - I can just multiply by 10 result in cycle.
I changed this part, thank you!